DESCRIPTION

```diff
@@ -12,6 +12,11 @@ Authors@R:
              role = c("aut", "cre"),
              email = "alexpghayes@gmail.com",
              comment = c(ORCID = "0000-0002-4985-5160")),
+      person(given = "Grant",
```

Comment:
Would you mind adding yourself at the end of the list rather than the beginning? That's what people have been doing so far and I'd like to stick with it for the time being.